sentry-android-core/src/main/java/io/sentry/android/core/AnrIntegration.java

```diff
@@ -0,0 +1,53 @@
+package io.sentry.android.core;
+
+import static io.sentry.core.ILogger.logIfNotNull;
+
+import io.sentry.core.IHub;
+import io.sentry.core.Integration;
+import io.sentry.core.SentryLevel;
+import io.sentry.core.SentryOptions;
+import io.sentry.core.exception.ExceptionMechanismThrowable;
+import io.sentry.core.protocol.Mechanism;
+
+public final class AnrIntegration implements Integration {
+  private static ANRWatchDog anrWatchDog;
+
+  @Override
+  public void register(IHub hub, SentryOptions options) {
+    register(hub, (SentryAndroidOptions) options);
```

Comment:
that's ok, we could also make `Integration<T extends SentryOptions>` or don't we want that?

Reply:
The problem is that we keep them on a `List<Integration>` and each integration now would close the generic paramter with something different and would be a problem to have `List<Integration<TOptions>>` since they'd differ